pinot-transport/src/main/java/com/linkedin/pinot/routing/HelixExternalViewBasedRouting.java

```diff
@@ -176,6 +178,9 @@ public void markDataResourceOnline(String tableName, ExternalView externalView,
     if (externalView == null) {
       return;
     }
+
+    long startTime = System.currentTimeMillis();
```

Comment:
On one large cluster that we have, latency for the routing table refresh is in the order of a few to several dozens seconds, so milliseconds should be fine. If this drops by two orders of magnitude or more, we can change the metric later to nanoseconds. The metric gets converted to seconds anyway, so this would not be a breaking change.